src/Symfony/Component/DependencyInjection/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -137,6 +137,9 @@ public function testLegacyLoadServices()
         $this->assertEquals('getInstance', $services['factory_service']->getFactoryMethod());
     }
 
+    /**
+     * @group legacy
+     */
```

Comment:
You're right, moving this particular service to the legacy file makes indeed more sense.